pkg/agent/flowexporter/exporter/exporter.go

```diff
@@ -225,6 +225,14 @@ func (exp *FlowExporter) Run(stopCh <-chan struct{}) {
 	// Start the goroutine to poll conntrack flows.
 	go exp.conntrackConnStore.Run(stopCh)
 
+	if exp.nodeRouteController != nil {
+		// Wait for NodeRouteController to have processed the initial list of Nodes so that
+		// the list of Pod subnets is up-to-date.
+		if !cache.WaitForCacheSync(stopCh, exp.nodeRouteController.HasSynced) {
+			return
+		}
+	}
+
```

Comment:
Should we move this at the top of this function?

Reply:
I will move it, but I am not sure it makes a difference

Reply:
Actually it means delaying the call to `exp.podStore.Run(stopCh)` which in my opinion is not great.
Later on I can add another wait call for `podStore.HasSynced`, and I will have a single call to `cache.WaitForCacheSync`.
In the mean time, I kept things as they were.